.github/workflows/publish.yml

```diff
@@ -6,14 +6,58 @@ on:
       - 'v[0-9]+.[0-9]+.[0-9]+*'
 
 jobs:
-  publish:
-    name: Publish to VS Code Marketplace
-    runs-on: ubuntu-latest
+  build:
+    name: Package
+    strategy:
+      matrix:
+        include:
+          - os: windows-latest
+            platform: win32
+            arch: x64
+            npm_config_arch: x64
+            target: win32-x64
+          - os: windows-latest
+            platform: win32
+            arch: ia32
+            npm_config_arch: ia32
+            target: win32-ia32
+          # - os: windows-latest
+          #   platform: win32
+          #   arch: arm64
+          #   npm_config_arch: arm
+          #   target: win32-arm64
```

Comment:
I remember you mentioning some issues with windows/arm64, what's the latest on this topic?

Reply:
I did not attempt to run this again on the test repo. Was too busy trying to get the whole thing working. Will circle back on this

Reply:
With the new matrix approach, this builds an arm distributable